hypothesis-python/scripts/basic-test.sh

```diff
@@ -27,6 +27,8 @@ pip install ".[dpcontracts]"
 $PYTEST tests/dpcontracts/
 pip uninstall -y dpcontracts
 
+# use pinned redis version instead of inheriting from fakeredis
+pip install "$(grep '^redis==' ../requirements/coverage.txt)"
 pip install "$(grep 'fakeredis==' ../requirements/coverage.txt)"
```

Comment:
Not proposing to change this now that it's working, but I think we could also use `-c requirements/coverage.txt` to inherit the pins as version constraints?  Maybe next time this comes up...